### PR TITLE
Fix packaging script files for CLI

### PR DESCRIPTION
**Pull request by b-butler**
## Description
Adds the necessary `__init__.py` file.

## Motivation and Context
Version 0.26.0 is broken when installing.

__Resolves__: #763

## Checklist:
- [x] I am familiar with the [Contributing Guidelines](https://github.com/glotzerlab/signac-flow/blob/main/CONTRIBUTING.md).
- [x] I agree with the terms of the [Contributor Agreement](https://github.com/glotzerlab/signac-flow/blob/main/ContributorAgreement.md).
- [x] My name is on the [list of contributors](https://github.com/glotzerlab/signac-flow/blob/main/contributors.yaml).
- [x] The changes introduced by this pull request are covered by existing or newly introduced tests.
- [x] The [package documentation](https://github.com/glotzerlab/signac-flow/tree/main/doc) and [framework documentation](https://docs.signac.io/) in [signac-docs](https://github.com/glotzerlab/signac-docs) are up to date with these changes.
- [x] I have updated the [changelog](https://github.com/glotzerlab/signac-flow/blob/main/changelog.txt) and added any related issue and pull request numbers for future reference.

### CODE REVIEW

**Review by mikemhenry — APPROVED** [review]

File: changelog.txt [hunk 1]
```diff
@@ -8,13 +8,17 @@ The numbers in brackets denote the related GitHub issue and/or pull request.
...
 
-[0.26.1] -- not yet released
-----------------------------
+[0.26.1] -- 2023-xx-xx
+----------------------
 
 Fixed
 +++++
 
+<<<<<<< HEAD
```
Comment (mikemhenry) [inline changelog.txt:17]:
```suggestion
```

File: changelog.txt [hunk 2]
```diff
@@ -8,13 +8,17 @@ The numbers in brackets denote the related GitHub issue and/or pull request.
...
+- Packaging of files in ``flow.scripts`` used in CLI (#765).
+=======
```
Comment (mikemhenry) [inline changelog.txt:19]:
```suggestion
```

File: changelog.txt [hunk 3]
```diff
@@ -8,13 +8,17 @@ The numbers in brackets denote the related GitHub issue and/or pull request.
...
 - Schedule 56 cores per node on OLCF Frontier (#764).
+>>>>>>> main
```
Comment (mikemhenry) [inline changelog.txt:21]:
```suggestion
```

**Review by mikemhenry — CHANGES_REQUESTED** [review]
Looks like a merge got botched

**Comment by b-butler** [comment]
@mikemhenry thanks for the catch.

File: flow/scripts/__init__.py [hunk 4]
```diff
@@ -0,0 +1,4 @@
+# Copyright (c) 2023 The Regents of the University of Michigan
+# All rights reserved.
+# This software is licensed under the BSD 3-Clause License.
+"""Import command line scripts."""
```
Comment (bdice) [inline flow/scripts/__init__.py:4]: Can we add very basic CLI tests? Even just `flow --help`. See signac for an example of this.

**Review by bdice — CHANGES_REQUESTED** [review]
Would like to request tests before merging.

File: flow/scripts/__init__.py (hunk 4 above)
Comment (b-butler) [reply flow/scripts/__init__.py:4]: We have these tests see https://github.com/glotzerlab/signac-flow/blob/main/tests/test_shell.py.

My guess is that tests are run in the repo directory which has the file but the install doesn't due to setuptools package detection requiring an `__init__.py`.

Comment (bdice) [reply flow/scripts/__init__.py:4]: I see. I'll approve this.

**Review by bdice — APPROVED** [review]

File: flow/scripts/__init__.py (hunk 4 above)
Comment (joaander) [reply flow/scripts/__init__.py:4]: This is why I run tests on installed packages without any source directory. Too many issues like this can occur.

**Pull request merged**